engine/graphics/src/vulkan/graphics_vulkan.cpp

```diff
@@ -1379,7 +1379,9 @@ namespace dmGraphics
         context->m_CurrentFrameInFlight = (context->m_CurrentFrameInFlight + 1) % context->m_NumFramesInFlight;
         context->m_FrameBegun           = 0;
 
-        dmPlatform::SwapBuffers(context->m_Window);
+#if defined(ANDROID) || defined(DM_PLATFORM_IOS)
+        dmPlatform::SwapBuffers(((VulkanContext*) context)->m_Window);
+#endif
```

Comment:
Readding what was removed from NativeSwapBuffers() in pr:

https://github.com/defold/defold/pull/9361/files#diff-2d2993beb5789c6fad6d72dd2b687547e9221ace2f49faa34173e7a2bc8603cc